### ISSUE TITLE
Missing translations in household heating technologies

### ISSUE THREAD

**Issue by MartLubben** [quintel/etmodel#3362]
![image](https://user-images.githubusercontent.com/33415156/83651436-c2907000-a5b9-11ea-9c33-44ea9dacc4ba.png)

**Comment by redekok** [comment]
Is this on BETA or PRO?

I would think this issue was fixed on BETA by https://github.com/quintel/etmodel/issues/3350?

**Comment by marliekeverweij** [comment]
Unfortunately not, I do not know why. @noracato do you maybe have time to look into this?

**Comment by noracato** [comment]
Yes, I thought I fixed this already in #3351. But now I see I forgot to put `nl:` above the [dutch translation](https://github.com/quintel/etmodel/blob/master/config/locales/interface/input_elements/nl_common.yml)... So I guess only the english one works. Sorry about that!
